ibm_randori/info.json

```diff
@@ -0,0 +1,653 @@
+{
+  "name": "ibm-randori",
+  "label": "IBM Randori",
+  "version": "1.0.0",
+  "description": "IBM Randori is an attack surface management SaaS that monitors internal and external attack surfaces for unexpected changes, blind spots, misconfigurations, and process failures. This connector facilitates automated operations to fetch network, target, service etc.",
+  "publisher": "Fortinet",
+  "cs_approved": false,
+  "cs_compatible": true,
+  "category": "Attack surface management",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "ingestion_supported": false,
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "title": "Server URL",
+        "description": "Service based URI to which you will connect and perform the automated operations.",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "server_url",
+        "value": "https://app.randori.io",
+        "tooltip": "Service based URI to which you will connect and perform the automated operations."
+      },
+      {
+        "title": "API Token",
+        "description": "API token configured for your account for using the IBM Randori API.",
+        "tooltip": "API token configured for your account for using the IBM Randori API.",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "password",
+        "name": "api_token"
+      },
+      {
+        "title": "Verify SSL",
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. <br />By default, this option is set as True.",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_ips",
+      "title": "Get IP Objects",
+      "description": "Retrieves IP objects based on the query, limit, offset parameter you have specified.",
+      "category": "investigation",
+      "annotation": "get_ips",
+      "output_schema": {
+        "count": "",
+        "data": "",
+        "offset": "",
+        "total": ""
+      },
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Query",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "q",
+          "tooltip": "Specify the query to retrieve the list of IP objects. e.g., {\n  \"condition\": \"AND\",\n  \"rules\": [\n    {\n      \"field\": \"table.confidence\",\n      \"operator\": \"greater_or_equal\",\n      \"value\": 60\n    },\n    {\n      \"field\": \"table.target_temptation\",\n      \"operator\": \"greater_or_equal\",\n      \"value\": 15\n    }\n  ]\n}",
```

Comment:
Add the example values in the values field, Since it becomes difficult for user to copy the format from tooltip.